plutus-use-cases/test/Spec/Governance.hs

```diff
@@ -32,17 +32,17 @@ import Test.Tasty.HUnit qualified as HUnit
 tests :: TestTree
 tests =
     testGroup "governance tests"
-    [ checkPredicate "vote all in favor, 2 rounds - SUCCESS"
+    [ checkPredicateOptions (defaultCheckOptions & allowBigTransactions) "vote all in favor, 2 rounds - SUCCESS"
         (assertNoFailedTransactions
         .&&. dataAtAddress (Scripts.validatorAddress $ Gov.typedValidator params) (maybe False ((== lawv3) . Gov.law) . listToMaybe))
         (doVoting 10 0 2)
 
-    , checkPredicate "vote 60/40, accepted - SUCCESS"
+    , checkPredicateOptions (defaultCheckOptions & allowBigTransactions) "vote 60/40, accepted - SUCCESS"
```

Comment:
Should `allowBigTransactions` be a default option?

Reply:
No, because that could lead to somebody concluding their code works on Mainnet while it doesn't.